.ci/scripts/run-gradle.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 # drop page cache and kernel slab objects on linux
 [[ -x /usr/local/sbin/drop-caches ]] && sudo /usr/local/sbin/drop-caches
+
+GRADLE_USER_HOME="$HOME/.gradle"
```

Comment:
Why is this needed?

Reply:
It honestly might not be, let me take it out and make sure everything is still good. I think it's left over from when I was debugging an issue that ended up being something else.

Reply:
Still good: https://buildkite.com/elastic/brianseeders-elasticsearch-sandbox/builds/134

I'll address it if it pops back up when I'm back to testing other pipelines